.circleci/config.yml

```diff
@@ -0,0 +1,58 @@
+version: 2  # use CircleCI 2.0 instead of CircleCI Classic
+jobs:  # basic units of work in a run
+  build:  # runs not using Workflows must have a `build` job as entry point
+    parallelism: 1  # run only one instance of this job in parallel
+    docker:  # run the steps with Docker
+      - image: circleci/elixir:latest  # ...with this image as the primary container; this is where all `steps` will run
```

Comment:
I would say to tag this with specific version

Reply:
Makes sense, thanks!